pubspec.lock

```diff
@@ -5,18 +5,18 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      sha256: "0c80aeab9bc807ab10022cd3b2f4cf2ecdf231949dc1ddd9442406a003f19201"
+      sha256: "3444216bfd127af50bbe4862d8843ed44db946dd933554f0d7285e89f10e28ac"
       url: "https://pub.dev"
     source: hosted
-    version: "52.0.0"
+    version: "50.0.0"
```

Comment:
Hmm interesting that most of these were downgraded... Do you need to update something?

Reply:
@tonidero  I had an old version of flutter when I started this so I think it downgraded stuff because of that... I just reset this file from `main` and now its just minor upgrades to some things 😇